src/game/server/ddracecommands.cpp

```diff
@@ -827,3 +827,50 @@ void CGameContext::ConDumpAntibot(IConsole::IResult *pResult, void *pUserData)
 	CGameContext *pSelf = (CGameContext *)pUserData;
 	pSelf->Antibot()->Dump();
 }
+
+void CGameContext::ConDumpLog(IConsole::IResult *pResult, void *pUserData)
+{
+	CGameContext *pSelf = (CGameContext *)pUserData;
+	int LimitSecs = MAX_LOG_SECONDS;
+	if(pResult->NumArguments() > 0)
+		LimitSecs = pResult->GetInteger(0);
+
+	if(LimitSecs < 0)
+		return;
+
+	for(auto &m_aLog : pSelf->m_aLogs)
+	{
+		if(!m_aLog.m_Timestamp)
+			continue;
+
+		int Seconds = (time_get() - m_aLog.m_Timestamp) / time_freq();
+		if(Seconds > LimitSecs)
+			continue;
+
+		char aBuf[256];
+		if(m_aLog.m_Server)
+			str_format(aBuf, sizeof aBuf, "%s, %i seconds ago", m_aLog.m_aDescription, Seconds);
+		else
+			str_format(aBuf, sizeof aBuf, "%s, %i seconds ago < addr=<{%s}> name='%s' client=%i",
```

Comment:
I'd suggest to use the conventional `%d` for integer type.

```suggestion
			str_format(aBuf, sizeof aBuf, "%s, %d seconds ago", m_aLog.m_aDescription, Seconds);
		else
			str_format(aBuf, sizeof aBuf, "%s, %d seconds ago < addr=<{%s}> name='%s' client=%d",
```